contributed_definitions/nyaml/NXsample_history.yaml

```diff
@@ -0,0 +1,17 @@
+category: base
+doc: |
+  A set of physical processes that occurred to  the sample prior/during experiment.
+  Ideally, a full report of the previous operations, in the format of NXphysical_process.
+  Alternatively, notes allows for additional descriptors in any format.
+type: group
+NXsample_history(NXobject):
+  (NXphysical_process):
+    doc: |
+      Any process that was performed on the sample prior or during the experiment.
+      Examples includ 

```

Comment:
The timeline in principle is encoded in the timestamps in the NXphysical_process instances. Should we also still provide a naming scheme fro the instances of the classes that encodes the succession of events?